src/cr/cube/matrix.py

```diff
@@ -975,11 +1029,11 @@ def pruned(self):
 
     @lazyproperty
     def pvals(self):
-        return np.array([np.nan] * len(self._matrix.columns))
+        return self._pvals
```

Comment:
These are absolutely the same between `_InsertionRow` and `_InsertionColumn`. They should just be moved one level up and public.